doc/changes/devel/12978.other.rst

```diff
@@ -0,0 +1 @@
+Fix a mistake in :ref:`_tut-artifact-regression` where the wrong regression coefficients were applied, by :newcontrib:`Jacob Phelan`.
```

Comment:
```suggestion
Fix a mistake in :ref:`tut-artifact-regression` where the wrong regression coefficients were applied, by :newcontrib:`Jacob Phelan`.
```


I think this should fix the CI error's (as seen [here](https://app.circleci.com/pipelines/github/mne-tools/mne-python/25820/workflows/0950c4e9-314f-4ebc-ad9f-de8d02d9a386/jobs/69800?invite=true#step-143-0_106) )!

Reply:
Thanks @JacPhe ! As far as I can tell I _think_ the current CI [error's](https://app.circleci.com/pipelines/github/mne-tools/mne-python/25822/workflows/e8331376-7010-49b5-bb5d-7c588e8079e8/jobs/69805?invite=true#step-107-26201_30) are unrelated to the changes in this PR. Let's wait for the devs to take a look.